### ISSUE TITLE
Link to curriculum onboarding playlist

### ISSUE THREAD

**Issue by maneesha** [carpentries/docs.carpentries.org#282]
[Item 5 in the Curriculum onboarding materials](https://docs.carpentries.org/resources/curriculum/curriculum_onboarding.html#for-curriculum-developers) directs users to a Youtube playlist but the link actually goes to a specific video.  

Should this link to this [playlist](https://www.youtube.com/playlist?list=PLXLapl_LKb4e73Vf2e3rS2q2TDJ7oh_DX) instead?

**Comment by tobyhodges** [comment]
Good catch @maneesha Yes please update the link to point to the playlist as you suggested.

**Comment by maneesha** [comment]
Fixed with #326
